code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -456,6 +456,7 @@
 	projectilesound = 'sound/effects/hit_on_shattered_glass.ogg'
 	ranged = 1
 	range = 13
+	ranged_cooldown_cap = 1
```

Comment:
already set, above, where the projectiletype is set; this may be intentional; not sure.